lib/active_record/virtual_attributes/virtual_fields.rb

```diff
@@ -190,10 +190,8 @@ def grouped_records(orig_association, records, polymorphic_parent)
 
               case association
               when Symbol, String
-                # 4/24/20 we want to revert #67 once we handle all these error cases in our codebase.
                 reflection = record.class._reflect_on_association(association)
-                display_virtual_attribute_deprecation("#{record.class.name}.#{association} does not exist") if !reflection && !polymorphic_parent
-                next if !reflection || !record.association(association).klass
+                next if polymorphic_parent && !reflection || !record.association(association).klass
```

Comment:
Interesting.  So, now, we'd do `next` for a nil/false `polymorphic_parent`.  Previously, was this a bug?

Reply:
This was answered below in https://github.com/ManageIQ/activerecord-virtual_attributes/pull/97#issuecomment-1010195099